trillian/ctfe/handlers.go

```diff
@@ -109,6 +109,8 @@ var (
 	rspLatency       monitoring.Histogram // logid, ep, rc => value
 )
 
+var errInternalServerError = errors.New("Internal Server Error")
```

Comment:
I think we need something blander/more vague for the error -- if a client submits something invalid (e.g. first > second) we don't want an error message that makes it sound like it's our fault.  Maybe something like "An error occurred processing this request" ?